helm-charts/prometheus/values.yaml

```diff
@@ -0,0 +1,24 @@
+defaultRules:
+  create: false
```

Comment:
```suggestion
```

Let's keep the default rules for now. We do not what effect the have on the deployment.